### DISCUSSION TITLE
Actions V2 - are the requests to the target authorized

### DISCUSSION THREAD

**Discussion by GrolarDan** [zitadel/zitadel#8639] [Q&A] [answered]
Hi,

when I have an API (target) I want to let only authorized requests go through.
Are the requests authorized and how?

Thanks

**Comment by hifabienne** [comment]
@adlerhurst @stebenz can you help here?

**Reply by adlerhurst** [reply, marked as answer]
hi @GrolarDan 

Thats currently open. See https://github.com/zitadel/zitadel/issues/7924 for more information.

**Reply by GrolarDan** [reply]
Thanks for response.
